src/js/vre/chapter36/config/form.js

```diff
@@ -141,27 +151,64 @@ const formConfig = {
             }
           }
         }
-      },
-      veteranInformation: {
-        title: 'Veteran Information',
-        pages: {
-        }
-      },
-      additionalInformation: {
-        title: 'Additional Information',
-        pages: {
-        }
-      },
-      militaryHistory: {
-        title: 'Military History',
-        pages: {
-        }
-      },
-      contactInformation: {
-        title: 'Contact Information',
-        pages: {
+      }
+    },
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
+          title: 'Veteran Inforation',
+          path: 'veteran-information',
+          uiSchema: {
+            veteranFullName: fullNameUI,
+            veteranDateOfBirth: currentOrPastDateUI('Date of birth'),
+            veteranSocialSecurityNumber: ssnUI,
+            veteranVaFileNumber: {
```

Comment:
Should file number and ssn be conditionally required if the other is not provided? e.g.
https://github.com/department-of-veterans-affairs/vets-website/blob/3ab5bf5632c04642063071d2f96a8e39833854cd/src/js/pensions/config/form.js#L218-L224

Reply:
We can check with @stacyhlinn, but I think the stakeholder wanted it this way for some reason.

Reply:
Not sure. The design has the SSN required. https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7137
Is the general approach to require at least one?

Reply:
Generally, yes, in some form. We have a the personId field where there's a checkbox that reveals file number and on other forms we've just required one or the other. I'd want to verify with Stacy what we're supposed to do here.

Reply:
Merged but added a question to the linked ticket.